src/main/java/EPparser/CommandParser.java

```diff
@@ -0,0 +1,237 @@
+package EPparser;
+
+import commands.*;
+import Execution.CommandStack;
+import MovieUI.Controller;
+import MovieUI.MovieHandler;
+import wrapper.CommandPair;
+import commands.COMMANDKEYS;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+
+public class CommandParser {
+    /**
+     * Entry point to command parser Class
+     *
+     * @param command command that was entered by the user
+     */
+    public static void parseCommands(String command , Controller UIController) throws IOException {
```

Comment:
Javadoc comments for method must start with a singular verb. Also, the 1st sentence of the comment should summarize the end-result of this method.

Same for those below.